건우/Week2_문자열뒤집기.js

```diff
@@ -0,0 +1,9 @@
+function solution(my_string) {
+    let answer = my_string.split('')
+
+    answer = answer.reverse()
+
+    answer = answer.join('')
+   
+   return console.log(answer);
```

Comment:
```suggestion
    return my_string.split('').reverse().join('');
```
해당 문제는 이런 방식으로도 풀 수 있어요. 자바스크립트에서는 연산 결과를 이어나갈 수 있습니다!